src/components/hv-select-single/index.js

```diff
@@ -46,16 +46,51 @@ export default class HvSelectSingle extends PureComponent<HvComponentProps> {
     const newElement = element.cloneNode(true);
     const options = newElement.getElementsByTagNameNS(
       Namespaces.HYPERVIEW,
-      'option',
+      LOCAL_NAME.OPTION,
     );
+
+    const isCumulative: boolean =
+      newElement.getAttribute('cumulative') === 'true';
+
+    // Calculate the index of the selected index. This is
+    // used in the cumulative mode, to render the preceding
+    // options as selected.
+    let selectedIndex: ?number = null;
+    for (let i = 0; i < options.length; i += 1) {
```

Comment:
should we skip this loop when `isCumulative` is `false` ?

Reply:
yes, I'll make that change.